std_msgs/Header header
PathTrace[] strokes
